.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "search.exclude": {
     "**/.sst": true
-  }
+  },
+  "typescript.preferences.importModuleSpecifier": "non-relative"
```

Comment:
This is a side fix for VSCode to use the `@/...` imports instead of relative ones